bin/stellar-core-backup

```diff
@@ -17,20 +18,25 @@ OptionParser.new do |opts|
     options[:list] = l
   end
   opts.on("-r", "--restore", "offline restore, uses latest backup") do |r|
-    options[:restore] = r
+    options[:type] = 'restore'
+  end
+  opts.on("-R", "--rm", "auto-clean buckets directory") do |r|
```

Comment:
Although namespaced, it might be clearer to have a unique `element` iterator instead of reusing `r` for each option in optparse blocks

Reply:
Fixed in latest code.